From e11498cece444b8f37363fd08322e87429d79e1c Mon Sep 17 00:00:00 2001
From: "Chrome Release Bot (LUCI)"
 <chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com>
Date: Sat, 25 Jun 2022 01:00:30 +0000
Subject: [PATCH 38/59] Incrementing VERSION to 102.0.5005.140

Change-Id: Ie76b68333f90b75b8ddcb541e3b0015b169e4a86
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3724132
Bot-Commit: Chrome Release Bot (LUCI) <chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/5005@{#1197}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 chrome/VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/VERSION b/chrome/VERSION
index 68428cc298bc..4bf645c073de 100644
--- a/chrome/VERSION
+++ b/chrome/VERSION
@@ -1,4 +1,4 @@
 MAJOR=102
 MINOR=0
 BUILD=5005
-PATCH=139
+PATCH=140
-- 
2.37.0

